presto-verifier/src/main/java/com/facebook/presto/verifier/framework/AbstractVerification.java

```diff
@@ -308,10 +303,8 @@ private VerificationResult concludeVerification(
                 sourceQuery.getName(),
                 partialResult.getStatus(),
                 partialResult.getSkippedReason(),
-                determinismAnalysis,
-                determinismAnalysis.isPresent() ?
-                        Optional.of(determinismAnalysisDetails) :
-                        Optional.empty(),
+                determinismAnalysisDetails.map(DeterminismAnalysisDetails::getAnalysis),
+                determinismAnalysisDetails,
```

Comment:
Should we change ```VerifierQueryEvent``` to just take ```determinismAnalysisDetails``` now that it contains the ```determinismAnalysis``` as well?

Reply:
done